ios/FluentUI/Vnext/Design Token System/ListHeaderFooterTokens.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// HeaderFooter styles
+@objc public enum MSFListHeaderFooterStyle: Int, CaseIterable {
+    case header
+    case headerBolded
+}
+
+public class MSFListHeaderFooterTokens: MSFTokensBase, ObservableObject {
+    @Published public var backgroundColor: UIColor!
+    @Published public var textColor: UIColor!
+
+    @Published public var horizontalCellPadding: CGFloat!
+
+    @Published public var textFont: UIFont!
+
+    var style: MSFListHeaderFooterStyle!
+
+    public init(style: MSFListHeaderFooterStyle) {
+        self.style = style
+
+        super.init()
+
+        self.themeAware = true
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        updateForCurrentTheme()
+    }
+
+    public override func updateForCurrentTheme() {
+        let currentTheme = theme
+        let appearanceProxy = currentTheme.MSFListHeaderFooterTokens
+        
+        switch style {
+        case .header:
```

Comment:
If you change the YAML file to have 2 different style definitions, here is where you would assign the proper appearance proxy.